src/sbt-test/sbt-scalafix/scalafixOnCompileConfig/test

```diff
@@ -0,0 +1,37 @@
+# check if scalafix-on-compile is disabled
+
+> example/compile
+-> example/scalafix
+
+# check if scalafix-on-compile is enabled and uses .scalafix.conf
+
+> set scalafixOnCompile.in(ThisBuild) := true
+-> example/compile
+-> example/scalafix
+
+# check if scalafix-on-compile uses .scalafix-on-compile.conf
+#   and scalafix-explicit-run still uses .scalafix.conf
+
+> set scalafixOnCompileConfig.in(ThisBuild) := file(".scalafix-on-compile.conf"),
+-> example/scalafix
+-> example/scalafix --check RemoveUnused
+> example/compile
+## A statement incl. null has been removed by the precedence scalafix-on-compile.
+> example/scalafix
+
+# check if scalafix-on-compile uses .scalafix-on-compile.conf
+#   and scalafix-explicit-run uses .scalafix-explicit-run.conf
```

Comment:
can you add an assertion that verifies that? wouldn't the test still pass if line 25 was removed?

Reply:
> set scalafixConfig.in(ThisBuild) := Some(file(".scalafix-explicit-run.conf")),

line 25(above) is mandatory for line 26, since `.scalafix-explicit-run.conf` and `.scalafix.conf` have different DisableSyntax config from each other.